packages/react/src/patterns/sub-patterns/Card/__stories__/Card.stories.js

```diff
@@ -1,39 +1,48 @@
 import './index.scss';
-import { select, text, withKnobs } from '@storybook/addon-knobs';
+import {
+  select,
+  text,
+  withKnobs,
+  boolean,
+  object,
+} from '@storybook/addon-knobs';
 import { ArrowRight20 } from '@carbon/icons-react';
 import { Card } from '../';
 import React from 'react';
 import readme from '../README.md';
 import { storiesOf } from '@storybook/react';
 
+const props = {
+  cardProps: () => ({
+    heading: text('title (required)', 'Lorem ipsum dolor sit amet'),
+    type: text('type', 'link'),
+    copy: text('copy', ''),
+    cta: object('cta', {
+      type: 'local',
+      copy: '',
```

Comment:
Probably want to at least add copy as default.

Reply:
@photodow Will make note to address for that component.

Reply:
done, actually doing so made me realize I didn't use `LinkWithIcon` for the static type 😀